blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueTestResult.java

```diff
@@ -0,0 +1,44 @@
+package io.jenkins.blueocean.rest.model;
+
+import io.jenkins.blueocean.rest.Navigable;
+import org.kohsuke.stapler.WebMethod;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.verb.GET;
+
+public abstract class BlueTestResult extends Resource {
+
+    public static final String STATUS = "status";
+    public static final String DURATION = "duration";
+    public static final String NAME = "name";
+    public static final String ID = "id";
+    public static final String STDERR = "stderr";
+    public static final String STDOUT = "stdout";
+
+    public enum Status {
+        PASSED,
+        FAILED,
+        SKIPPED
+    }
+
+    @Exported(name = STATUS)
```

Comment:
I know this is a common pattern I've seen, but I don't quite get why we have statics for these